text/0000-rebase-immutable-image-ref.md

```diff
@@ -0,0 +1,110 @@
+# Meta
+[meta]: #meta
+- Name: Rebase by Image Digest Reference
+- Start Date: 2022-12-08
+- Author(s): [@joeybrown-sf](https://github.com/joeybrown-sf)
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a feature for the implementation team.
+
+Allow passing a digest reference as rebase target. 
+
+# Definitions
+[definitions]: #definitions
+
+An **image reference** refers to either a **tag reference** or **digest reference**.
+
+A **tag reference** refers to an identifier of form `<registry>/<repo>:<tag>` which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+
+A **digest reference**  refers to a [content addressable](https://en.wikipedia.org/wiki/Content-addressable_storage) identifier of form `<registry>/<repo>@<digest>` which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+
+
+# Motivation
+[motivation]: #motivation
+
+- This feature will support a workflow where the rebase executing agent only has the digest available to it.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define any new terminology.
+- Define the target persona: buildpack author, buildpack user, platform operator, platform implementor, and/or project contributor.
+- Explaining the feature largely in terms of examples.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to existing users and new users.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+First, let's get some context. Today, we can execute rebase by using **tag references** but not **digest references**.
+
+Here are some examples of valid rebase commands using **tag references**:
+1. `lifecycle rebase my-repo/foo`
```

Comment:
is it `rebase` or `rebaser`? I don't recall. I do know the binary location is `/cnb/lifecycle/rebaser`